engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
I actually think it's not only neater but more practical to have them parameters of the actor.
If you look at the Workflow Actor for example right now we have Actor messages case classes  a with never ending list of parameters, because the same actor is doing all the work and needs to handle every possible combination of messages and parameters + needs to maintain a complex mutable state data to know what to do when message A arrives from B but needs to wait for C to answer before processing it and in the meantime keep answering requests from D etc...
If you have smaller dedicated actors, pretty much all they need to know can be given as parameters which reduces their mutable data, which is good.